core/sql/ustat/hs_globals.cpp

```diff
@@ -3776,7 +3776,7 @@ NABoolean HSGlobalsClass::isAuthorized(NABoolean isShowStats)
        if (privs == NULL)
          {
            *CmpCommon::diags() << DgSqlCode(-1034);
-            authorized = FALSE;
+            return FALSE;
```

Comment:
The return was added because the next statement "if ( privs->hasSelectPriv() )" cores if privs is NULL.     But will change the code to not break the time.